utopia-remix/app/routes/login.tsx

```diff
@@ -0,0 +1,25 @@
+import { redirect, type LoaderFunctionArgs } from '@remix-run/node'
+import { ALLOW } from '../handlers/validators'
+import { handle } from '../util/api.server'
+import type { Params } from '@remix-run/react'
+import { auth0LoginURL } from '../util/auth0.server'
+
+export async function loader(args: LoaderFunctionArgs) {
+  return handle(args, {
+    GET: {
+      validator: ALLOW,
+      handler: handleLogin,
+    },
+  })
+}
+
+export async function handleLogin(req: Request, params: Params<string>) {
```

Comment:
I think that if the user is already logged in this should do nothing and return immediately (redirecting to the `redirectTo` maybe)

Reply:
so I think that usually in SaaS systems the `/login` route always takes you to the login page even if you are logged-in (for example if you want to change accounts, or have a new session). We can decide to do it differently here of course

Reply:
yeah, we should align with whatever pattern makes more sense at the moment – this is fine by me